### PR TITLE
fix: Can list and relist to all marketplaces

### PR DESCRIPTION
**Pull request by cbachmeier**
- Abstract away OS listing to BE endpoint
- Allow for relisting of X2Y2 assets
- Support new LR listing strategy
- Fixes bug with incorrectly formatted basis points (creator royalties)

Tested by being able to successfully list and relist to all 3 marketplaces

### CODE REVIEW

File: src/nft/queries/openSea/PostOpenSeaSellOrder.ts [hunk 1]
```diff
@@ -1,77 +1,24 @@
...
-  if (response.ok) {
-    return response
-  }
-
-  let result
-  let errorMessage
+    body,
+    signal: ac.signal,
+  })
+  const timeout = setTimeout(() => ac.abort(), 60 * 1000)
```
Comment (JackShort) [inline src/nft/queries/openSea/PostOpenSeaSellOrder.ts:13]: non-blocking: we could use `ms` for clarity

File: src/nft/utils/listNfts.ts [hunk 2]
```diff
@@ -238,14 +244,15 @@ export async function signListing(
...
         await signOrderData(provider, order)
         const payload: OrderPayload = {
           order: encodeOrder(order),
           isBundle: false,
           bundleName: '',
           bundleDesc: '',
-          orderIds: [],
-          changePrice: false,
+          orderIds: prevOrderId ? [prevOrderId] : [],
+          changePrice: Boolean(prevOrderId),
```
Comment (JackShort) [inline src/nft/utils/listNfts.ts:255]: non-blocking: instead of casting you could `!!`

**Review by JackShort — APPROVED** [review]
lgtm

File: src/nft/utils/listNfts.ts (hunk 2 above)
Comment (cbachmeier) [reply src/nft/utils/listNfts.ts:255]: Boolean is actually faster as of 2020

**Pull request merged**